### PR TITLE
doc: note about module names limit

### PR DESCRIPTION
**Pull request by lkppo**
Sic

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -1703,7 +1703,8 @@ fn main() {
 }
 ```
 
-* Module names should be short, under 10 characters.
+* Module names should be short, under 10 characters (limited by the windows MAX_PATH
+since modules use directory names).
```
Comment (spytheman) [inline doc/docs.md:1707]: why?
afaik windows folders names can be > 10 characters, since at least Windows NT?

Comment (spytheman) [reply doc/docs.md:1707]: From https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file : 
> In editions of Windows before Windows 10 version 1607, the maximum length for a path is MAX_PATH, which is defined as 260 characters. In later versions of Windows, changing a registry key or using the Group Policy tool is required to remove the limit. See Maximum Path Length Limitation for full details.

Comment (spytheman) [reply doc/docs.md:1707]: The 10 character suggestion is only a style guide imho, not a technical limitation.

Comment (lkppo) [reply doc/docs.md:1707]: Forget this PR. Limits could have their own section.

**Pull request closed without merging**